src/Confluent.Kafka/ErrorCode.cs

```diff
@@ -29,6 +29,26 @@ namespace Confluent.Kafka
     public enum ErrorCode
     {
         /// <summary>
+        ///     Value serialization error
+        /// </summary>
+        Local_ValueSerialization = -1004,
```

Comment:
The extra error codes are specific to librdkafka in that they won't be used by anything other than a library that wraps librdkafka - I can't think of any better place for them to be defined outside of confluent-kafka-dotnet. 

I don't mind too much if they are defined only in confluent-kafka-dotnet (and repeated in other libraries if necessary).

I also like `Client_` better.